File: MODUL-6/Bahasa C/PRAK603-2310817120014-ALIYARAFFANAURAAYU.c
#include <stdio.h>

int main (void)
{
    int b1, b2, x, isi ;
    scanf("%d %d", &b1, &b2) ;
    int a1[b1], a2[b2] ;
    if (b1 != b2) {
        printf ("\nJumlah tidak sama") ;
    }
    else {
        for (x = 0; x < b1; x++) {
            scanf ("%d", &isi) ;
            a1[x] = isi ;
        }
        for (x = 0; x < b2; x++) {
            scanf ("%d", &isi) ;
            a2[x] = isi ;
        }
        printf ("\n") ;
        for (x = 0; x < b1; x++) {
            printf ("%d ", a1[x] * a2[x]) ;
        }
    }
}

File: MODUL-6/Bahasa Python/PRAK603-2310817120014-ALIYARAFFANAURAAYU.py
b1, b2 = map(int, input().split())
i1 = list(map(int, input().split()))
i2 = list(map(int, input().split()))
a1 = [] ; a2 = []
if (b1 != b2) :
    print("\nJumlah tidak sama")
else :
    print(" ")
    for x in range(0, b1) :
        a1.append(i1[x])
    for x in range(0, b2) :
        a2.append(i2[x])
    for x in range(0, b1) :
        print(a1[x] * a2[x], end=' ')